macros/sql/surrogate_key.sql

```diff
@@ -2,7 +2,7 @@
 
 {%- if varargs|length >= 1 or field_list is string %}
 
-{%- do exceptions.warn("Warning: the `surrogate_key` macro now takes a single list argument instead of multiple string arguments. Support for multiple string arguments will be deprecated in a future release of dbt-utils.") -%}
+{%- do exceptions.warn("Warning: the `surrogate_key` macro now takes a single list argument instead of multiple string arguments. Support for multiple string arguments will be deprecated in a future release of dbt-utils. The " ~ model.package_name ~ "." ~ model.name ~ " model triggered this warning.") -%}
```

Comment:
I wouldn't hate something like this:
```suggestion
{%- set error_message = '
Warning: the `surrogate_key` macro now takes a single list argument instead of \
multiple string arguments. Support for multiple string arguments will be \
deprecated in a future release of dbt-utils. The {}.{} model triggered this warning. \
'.format(model.package_name, model.name) -%}

{%- do exceptions.warn(error_message) -%}
```

Reply:
Sure I can do it real quck.

Reply:
@clrcrl added 🙂